### PR TITLE
Mocking loggers (part 1, wandb)

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

wandb tests already involve mocking. 
Here we remove the install of wandb completely. 
This does not change anything for the user.

### CODE REVIEW

**Comment by awaelchli** [comment]
so when we mock and remove the wandb install for example, we lose the possibility to run the doctests.

**Review by Borda — APPROVED** [review]

File: requirements/extra.txt [hunk 1]
```diff
@@ -1,10 +1,5 @@
 # extended list of package dependencies to reach full functionality
 
-neptune-client>=0.4.109
-comet-ml>=3.1.12
-mlflow>=1.0.0
-test_tube>=0.7.5
```
Comment (awaelchli) [inline requirements/extra.txt:6]: @Borda these cannot yet be removed, because they are still not mocked, I will try to do them one by one in follow up

Comment (awaelchli) [reply requirements/extra.txt:6]: the extra.txt is used in CI build

Comment (Borda) [reply requirements/extra.txt:6]: I think that no the CI install from `pip install .["extra"]` right?

Comment (awaelchli) [reply requirements/extra.txt:6]: if you add loggers.txt to devel, will it install in ci or not?

Comment (Borda) [reply requirements/extra.txt:6]: ok, true, I ll revert...

Comment (awaelchli) [reply requirements/extra.txt]: thanks

**Review by SkafteNicki — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Pull request merged**